config/mail.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'driver' => env('MAIL_DRIVER', 'smtp'),
+    'driver' => env('MAIL_DRIVER', 'log'),
```

Comment:
@angaither it is in my .env, i just had the log as the default for development. I can reset this though, we should probably default to mandrill.